xml/deployment_prep_x86_boot.xml

```diff
@@ -40,7 +40,7 @@
    You can still perform the installation on the console without a working
    graphics mode. The text-based installation program offers the same
    functionality as the graphical version. Find some hints about navigation in
-   this mode in <xref linkend="sec.yast.navi.ncurses_modul"/>.
+   this mode in <xref linkend="sec.yast.text.navi_module"/>.
```

Comment:
* "navi" seems like a Germanism.
* "text" seems wrong -- why not use "ncurses" instead?

i.e, How about "sec.yast.ncurses.navigate"?